css/dark-mode.css

```
@@ -203,3 +204,17 @@ html.dark-mode ._9xq0 {
 html.dark-mode .x14hiurz {
 	background-color: transparent !important;
 }
+
+/* Attachement Background Color in Chat Box*/
+/* TODO: Remove when fixed by fb, since #d6d9dd is too bright here */
+/* Hover overlay is same style as add attachment button */
+html.dark-mode .x1ybostu {
+    background-color: var(--hover-overlay);
+}
+
+/* Recipient Pills in new message form*/
+/* TODO: Remove when fixed by fb, since #f3f3f4 is too bright here */
+/* It seems FB uses shades of blue here */
+html.dark-mode .x4noic3{
+	background-color: transparent;
```

This one looks better with secondary button background color imho. You can then clearly see the pill shape.
```suggestion
	background-color: var(--secondary-button-background);
```